rDNA/R/rDNA.R

```diff
@@ -7902,19 +7945,19 @@ dna_barplot <- function(connection,
   dta2 <- dta2[order(dta2$absFrequency, decreasing = TRUE), ]
   # replicate order of dta2$of to dta
   dta$of <- factor(dta$of, levels = rev(dta2$of))
-  # get bar colours
-  if (colours) {
+  # get bar colors
+  if (colors) {
     if (!"statementType" %in% names(dots)) {
       dots$statementType <- "DNA Statement"
     }
     col <- dna_getAttributes(connection = connection, statementType = dots$statementType,
                              variable = of, values = NULL)
-    dta$colour <- as.character(col$color[match(dta$of, col$value)])
-    dta$text_colour <- "black"
-    dta$text_colour[sum(grDevices::col2rgb(dta$colour) * c(299, 587, 114)) / 1000 < 123] <- "white"
+    dta$color <- as.character(col$color[match(dta$of, col$value)])
+    dta$text_color <- "black"
+    dta$text_color[sum(grDevices::col2rgb(dta$color) * c(299, 587, 114)) / 1000 < 123] <- "white"
   } else {
-    dta$colour <- "white"
-    dta$text_colour <- "black"
+    dta$color <- "white"
+    dta$text_color <- "black"
   }
   if (binary) {
```

Comment:
Partial matching?

Reply:
Sorry, I don't know what you mean.

Reply:
`isTRUE(binary)`?

Reply:
`binary` is set via `binary <- all(dta$agreement %in% c(0, 1))` so it can only be `TRUE` or `FALSE`, as far as I know.